pkg/users/users.js

```diff
@@ -91,7 +91,14 @@ function AccountsPage() {
         const handleShadow = cockpit.file("/etc/shadow", { superuser: "try" });
         handleShadow.watch(() => debouncedGetLoginDetails(), { read: false });
 
-        const handleLogindef = cockpit.file("/etc/login.defs");
+        let handleLogindef;
+        try {
+            await cockpit.spawn(["test", "-e", "/etc/login.defs"], { err: "ignore" });
```

Comment:
This is a bit expensive -- could you please do this instead:
```js
import { fsinfo } from "cockpit/fsinfo";

try {
    await fsinfo("/etc/login.defs"), []);
    /* file exists */
} catch (ex) {
   /* file does not exist
}
```

Thanks!

Reply:
And should we be wary of more /etc/ config files then? ie:

/etc/default/grub
/etc/kdump.conf
/etc/hostname
/etc/sysconfig/kdump
/etc/services (firewall page)

I assume this is for [hermetic usr configs](https://github.com/uapi-group/specifications/issues/76)